index.html

```diff
@@ -0,0 +1,164 @@
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Humita" target="_blank"><img class="ingredient-img" src="images/vectors/corn.png" alt="peruvian ingredient corn"></a>
+            <a rel="noopener" href="https://en.wikipedia.org/wiki/Pisco_sour" target="_blank"><img class="ingredient-img" src="images/vectors/lemon.png" alt="peruvian ingredient lemon"></a>
+            <a rel="noopener" href="https://es.wikipedia.org/wiki/Archivo:Ensalada_Palta.JPG" target="_blank"><img class="ingredient-img" src="images/vectors/avocado.png" alt="peruvian ingredient avocado"></a>
+            <a rel="noopener" href="https://www.aracari.com/blog/stories/a-guide-to-peruvian-potatoes/" target="_blank"><img class="ingredient-img" src="images/vectors/potatoes.png" alt="peruvian ingredient potatoes"></a>
+        </div> 
+
+
+        <div id="heading-cont" class="heading">
+            <h2 id="main-subtitle" class="main-subtitle">International Culinary Summit</h2>
+            <h1 id="main-title" class="main-title">FLAVORS OF PERU 2022</h1>
+        </div>
+
+        <div id="description-cont">
+            <p id="description" class="description">A closer look at the evolution of Peruvian cuisine, in hand with 5 of the country's most laureated representatives. The ultimate celebration of the senses.</p>
+        </div>
+
+        <div id="dates-venue">
+            <h2 id="dates" class="dates">2022/07/27(WED) - 29(FRI)</h2>
+            <h4 id="venue" class="venue">@ Admiralspalast Berlin, DE</h4>
+        </div>
+
+    </header>
+
+    <section id="program" class="program-container">
+
+        <h2 id="program-main">Main Program</h2>
+
+        <hr class="separator">
+
+        <div id="program-cards-container">
+            <div  class="program-card">
+                <img class="card-icon" src="images/icons/keynote.png" alt="">
+                <p class="card-title">Lecture</p>
+                <p class="card-description">Listen to the event's keynote speakers about their culinary industry journey and learn from their experiences.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/dish.png" alt="">
+                <p class="card-title">Tasting</p>
+                <p class="card-description">Enjoy yourself with an exquisite showcase of Peru's most dishes and beverages.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/chat.png" alt="">
+                <p class="card-title">Forum</p>
+                <p class="card-description">Have the chance to share your thoughs, questions and experiences not only with the speakers of the event, but with colleagues from all around the world.</p>
+            </div>
+
+            <div class="program-card">
+                <img class="card-icon" src="images/icons/chef.png" alt="">
```

Comment:
- [ ] Please add an alt description for the image, this is important for best HTML practices. 🐤